platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -201,6 +201,10 @@ event_processor:
 #   If the type is `noncustodial`,
 #   - domains: (required) the domain of the client.
 #   - callback_url: (optional) the URL of the client's callback API endpoint
+#   - callback_url_sep6: (optional) the URL of the client's SEP-6 callback API endpoint.
```

Comment:
How does the `callback_url` and `callback_url_sep6` and others interact? Can we document here?

Reply:
It's documented under the `custodial` client config. I decided not to repeat the full descriptions here since we don't do this for the other fields.